www/_posts/2016-05-24-tools-release.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: post
+author:
+    name: Steve Gill
+    url: https://twitter.com/stevesgill
+title:  "Tools Release"
+categories: news
+tags: release tools
+---
+
+New updates of `cordova-cli`, `cordova`, `plugman` and `cordova-common` are now live! We have also released the first version of `cordova-fetch`, a module used by `cordova-lib` to fetch plugins and platforms via `npm install`. 
+
+* [cordova-lib@6.2.0](https://www.npmjs.org/package/cordova-lib)
+* [cordova@6.2.0](https://www.npmjs.org/package/cordova)
+* [plugman@1.3.0](https://www.npmjs.org/package/plugman)
+* [cordova-common@1.3.0](https://www.npmjs.org/package/cordova-lib)
+* [cordova-fetch@1.0.0](https://www.npmjs.org/package/cordova-lib)
+
+Release Highlights:
+
+* [CB-9858](https://issues.apache.org/jira/browse/CB-9858) added `--fetch` option
+* [Telemetry](https://github.com/apache/cordova-cli/pull/247) Added telemetry to cordova-cli to collect data for data driven development. Goal is use data to help guide the development of cordova. It is auto opt-out. Type `cordova telemetry -h` for more info. 
```

Comment:
"auto opt-out" is unclear. Maybe "It is an opt-in feature that is off by default"?